third_party/2and3/typing_extensions.pyi

```diff
@@ -81,6 +81,10 @@ if sys.version_info >= (3, 5):
 
 if sys.version_info >= (3, 6):
     from typing import AsyncGenerator as AsyncGenerator
+    @runtime_checkable
```

Comment:
Why is this in the 3.6 block? From the linked PR it looks like it should exist in all versions except 3.5.0.

Reply:
Oops, I misremembered the guidance from `CONTRIBUTING.md` about how to handle micro versions. Repushed without the check.